tensorflow/python/estimator/export/export.py

```diff
@@ -245,16 +242,10 @@ class SupervisedInputReceiver(
   def __new__(cls, features, labels, receiver_tensors):
     # Both features and labels can be dicts or raw tensors.
     for input_vals, error_label in ((features, 'feature'), (labels, 'label')):
-      if input_vals is None:
-        raise ValueError('{}s must be defined.'.format(error_label))
-      if isinstance(input_vals, dict):
-        for name, tensor in input_vals.items():
-          _check_tensor_key(name, error_label=error_label)
-          _check_tensor(tensor, name, error_label=error_label)
-      else:
-        _check_tensor(input_vals, None, error_label=error_label)
-
-    receiver_tensors = _wrap_and_check_receiver_tensors(receiver_tensors)
+      _wrap_and_check_input_tensors(input_vals, error_label)
```

Comment:
Yes, it does convert raw tensors into a dict before the check, but the end result remains unaffected since it doesn't affect user code in any way. You can still pass raw tensors in.

Reply:
I'm a little nervous about using the side effect of the function here. The next helpful engineer might think, "Ah, the input tensors don't get replaced by the return. This must be a bug." And then they helpfully fix it, but change behavior. Can we just leave the input_vals section as is?

Reply:
Good point. I did feel a pang of "This doesn't seem too elegant" when replacing that one, but went with it anyway. 

At the same time, I feel there must be a better way to handle this than just not calling the function there, because that would just reintroduce the redundant code (something that this I intended to do as far as possible with this PR). 

Perhaps, a comment might suffice making it clear that dropping the assignment is a deliberate act? While you munch over it, I'll try to think of an alternative solution for this.

Reply:
A comment will probably suffice. Another option is to refactor just the checking, and call that from the wrapping fn, but that may be overkill here. Thanks, and sorry for the delay in response

Reply:
@karmel I've added a comment before the call.